### ISSUE TITLE
A BUG found in run.py

### ISSUE THREAD

**Issue by Silhouetti** [lilstrawberry/MIKT#4]
Dear author, I found a bug in run.py when I was about to reproduce your code.
In line 72 of run.py, the code "auc = metrics.roc_auc_score(labels, outputs)" seems to have missed importing metrics from sklearn.
I dont know if my judgement is right, but thank you for your wonderful work!

**Comment by lilstrawberry** [comment]
Thank you so much for pointing out this issue! 

I will update the code
